marlowe-playground-client/src/Home.purs

```diff
@@ -3,42 +3,87 @@ module Home where
+import Halogen.Css (classNames)
+import Halogen.HTML (a, button, div, h1, h2_, hr_, img, p_, span, span_, text)
 import Halogen.HTML.Events (onClick)
-import Halogen.HTML.Properties (classes, href, src, target)
+import Halogen.HTML.Properties (href, src, target)
 import MainFrame.Types (Action(..), ChildSlots, ModalView(..), State)
+import NewProject.Types as NewProject
+import Projects.Types (Lang(..))
 
 render :: forall m. State -> ComponentHTML Action ChildSlots m
 render state =
-  div [ classes [ ClassName "homepage-container", paddingBottom ] ]
-    [ div [ classes [ ClassName "marlowe-intro-container" ] ]
-        [ div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "What is Marlowe?" ]
-            , p_ [ text "Marlowe is special-purpose language for financial contracts on Cardano, allowing contracts to be written in the language of finance, rather than using a general-purpose language on the blockchain. Because it is special-purpose, it is easier to read, write and understand Marlowe contracts. It is also safer: some sorts of errors are impossible to write, and we can completely analyse contract behaviour without having to run a contract." ]
+  div [ classNames [ "flex", "flex-col", "items-center" ] ]
+    [ h1 [ classNames [ "font-semibold", "text-4xl", "my-16" ] ] [ text "Get started" ]
+    , div [ classNames [ "mb-6" ] ]
+        [ button
+            [ classNames (secondaryButton <> [ "mr-small", "w-56", "text-base" ])
+            , onClick ((const <<< Just <<< OpenModal) NewProject)
             ]
-        , div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "What is the Marlowe Playground?" ]
-            , p_ [ text "In the browser-based Marlowe Playground you can write Marlowe contracts, in a variety of different ways. Once a contract is written, you can analyse its behaviour, e.g. checking whether any payments made by the contract could conceivably fail. You can also step through how a contract will behave, simulating the actions of the participants, and read a comprehensive tutorial on Marlowe and the Playground." ]
-            ]
-        , div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "How does the playground work?" ]
-            , p_ [ text "Marlowe contracts can be built in different ways. You can write them as Marlowe text, but also use the Blockly visual programming tool to create contracts by fitting together blocks that represent the different components. Marlowe is embedded in JavaScript and Haskell, and so you can use features from them to help you to build Marlowe contracts more readably and succinctly." ]
+            [ text "Open existing project" ]
+        , button
+            [ classNames (primaryButton <> [ "ml-small", "w-56", "text-base" ])
+            , onClick ((const <<< Just <<< OpenModal) OpenDemo)
             ]
+            [ text "Open an example" ]
```

Comment:
These buttons don't have `cursor: pointer`. But I'm not sure why, because I thought that was included in all buttons by tailwind's reset. 😕

Reply:
we're not using tailwinds reset, we are using some reset css that was here from before... eventually we should move to tailwind's

Reply:
manually added for now.